Libraries/LibGUI/ListView.h

```diff
@@ -62,9 +62,11 @@ class ListView : public AbstractView {
     virtual void move_cursor(CursorMovement, SelectionUpdate) override;
     void move_cursor_relative(int steps, SelectionUpdate);
 
-private:
+protected:
     ListView();
+    virtual void paint_list_item(Painter& painter, int row_index, int painted_item_index);
```

Comment:
Per the coding style, please omit parameter names that are just the type name again (`Painter& painter` -> `Painter&` in this case)